libsyclinterface/source/dpctl_sycl_device_manager.cpp

```diff
@@ -52,17 +52,15 @@ std::string get_device_info_str(const device &Device)
     std::stringstream ss;
 
     ss << std::setw(4) << " " << std::left << std::setw(16) << "Name"
-       << Device.get_info<info::device::name>() << '\n'
+       << Device.get_info<info::device::name>() << std::endl
        << std::setw(4) << " " << std::left << std::setw(16) << "Driver version"
-       << Device.get_info<info::device::driver_version>() << '\n'
+       << Device.get_info<info::device::driver_version>() << std::endl
        << std::setw(4) << " " << std::left << std::setw(16) << "Vendor"
-       << Device.get_info<info::device::vendor>() << '\n'
+       << Device.get_info<info::device::vendor>() << std::endl
        << std::setw(4) << " " << std::left << std::setw(16) << "Profile"
-       << Device.get_info<info::device::profile>() << '\n'
```

Comment:
unless you really want to force a flush at this point, just adding \n is enough.

Reply:
I was not aware of the flushing peculiarity. I was thinking that on Windows ``std::endl`` would produce `"\n\r"`. My motivation was portability.

Reply:
Per https://stackoverflow.com/a/213977 I should revert it back. Will do.